crates/starknet-devnet-core/src/blocks/mod.rs

```diff
@@ -102,7 +109,7 @@ impl StarknetBlocks {
         to: Option<BlockId>,
     ) -> DevnetResult<Vec<&StarknetBlock>> {
         // used btree map to keep elements in the order of the keys
```

Comment:
```suggestion
        // used IndexMap to keep elements in the order of the keys
```

Reply:
Fixed in c44b87d57201e532a9aa0730be22f74d5fca4d1b